test/integration/instance.validations.test.ts

```diff
@@ -0,0 +1,552 @@
+import assert from 'node:assert';
+import { expect } from 'chai';
+import type {
+  InferAttributes,
+  CreationOptional,
+  InferCreationAttributes,
+  HasManyAddAssociationMixin,
+  BelongsToSetAssociationMixin,
+  HasMany,
+  BelongsTo,
+} from '@sequelize/core';
+import {
+  DataTypes,
+  ValidationError,
+  Model,
+} from '@sequelize/core';
+import {
+  sequelize,
+  getTestDialectTeaser,
+  getTestDialect,
+  beforeAll2,
+  disableDatabaseResetForSuite,
+} from './support';
+
+describe(getTestDialectTeaser('InstanceValidator'), async () => {
+  describe('#update', async () => {
+    it('should allow us to update specific columns without tripping the validations', async () => {
+      class User extends Model<InferAttributes<User>, InferCreationAttributes<User>> {
+        declare id: CreationOptional<number>;
+        declare username: string;
+        declare email: string;
+      }
+
+      User.init({
+        id: {
+          type: DataTypes.INTEGER,
+          primaryKey: true,
+          autoIncrement: true,
+        },
+        username: DataTypes.STRING,
+        email: {
+          type: DataTypes.STRING,
+          allowNull: false,
+          validate: {
+            isEmail: {
+              msg: 'You must enter a valid email address',
+            },
+          },
+        },
+      }, { sequelize });
+
+      await User.sync({ force: true });
+
+      const user = await User.create({ username: 'bob', email: 'hello@world.com' });
+
+      await User
+        .update({ username: 'toni' }, { where: { id: user.id } });
+
+      const user0 = await User.findByPk(1);
+      expect(user0?.username).to.equal('toni');
```

Comment:
We know it exists, so use non-null assertion instead of optional chaining

```suggestion
      expect(user0!.username).to.equal('toni');
```